src/controllers/aer_controller.hpp

```diff
@@ -1293,6 +1293,127 @@ void Controller::run_circuit(const Circuit &circ,
   }
 }
 
+void Controller::run_batched_circuits(const std::vector<Circuit> &circs,
+                             const Noise::NoiseModel &noise,
+                             const Method method,const json_t &config,
+                             Result &result) const 
```

Comment:
I believe that this method can be reduced.

Reply:
integrated and removed